extensions/azurecore/package.json

```diff
@@ -360,10 +360,10 @@
   "dependencies": {
     "@azure/arm-resourcegraph": "^4.0.0",
     "@azure/arm-subscriptions": "^3.0.0",
-    "@azure/storage-blob": "^12.6.0",
```

Comment:
Is this not being used anymore or something?

Reply:
Yarn auto-resolved it, so probably not required.

Reply:
`yarn install`? That shouldn't have happened, install doesn't modify the package.json.

Anyways we directly use storage-blob [here](https://github.com/Microsoft/azuredatastudio/blob/86c3f315f22be13e1456ead6423f3c9f1a8aea39/extensions/azurecore/src/azureResource/utils.ts) so please add this back.

Reply:
Strange it got removed - I've added it back!